app/src/main/res/values/strings.xml

```diff
@@ -41,7 +41,7 @@
     <!-- Content description for the recently saved bookmarks section on the home screen. -->
     <string moz:removedIn="100" name="recently_saved_bookmarks_content_description" tools:ignore="UnusedResources">Recently saved bookmarks</string>
     <!-- Title for the button which navigates the user to show all of their saved bookmarks. -->
-    <string name="recently_saved_show_all">Show all</string>
+    <string moz:removedIn="100" name="recently_saved_show_all" tools:ignore="UnusedResources">Show all</string>
```

Comment:
This respects the flow described in https://github.com/mozilla-mobile/fenix/wiki/Removing-strings.
Nice! 🏅

Reply:
Please make sure the argument for `moz:removedIn`, in this case "100" is in sync with the nightly version at the time of the merge.  If we'll have moved to "101" this will have to be updated to reflect that and in so prevent the string from actually being removed earlier than needed.

Reply:
Can we also reorder this 
```suggestion
    <string name="recently_saved_show_all" moz:removedIn="100" tools:ignore="UnusedResources">Show all</string>
```

Reply:
Sure, consider it done!